From: John Rigby <jrigby@freescale.com>
Date: Thu, 24 Aug 2006 12:13:48 -0600
Subject: [PATCH] ppc libavcodec bigendian

Define WORDS_BIGENDIAN in libavcodec for ppc.

Signed-off-by: John Rigby <jrigby@freescale.com>

---

 qtopia/src/3rdparty/libraries/libavcodec/bswap.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

fe2f39332febd2dc0378c28f3c3b8b7122cb0e10
diff --git a/qtopia/src/3rdparty/libraries/libavcodec/bswap.h b/qtopia/src/3rdparty/libraries/libavcodec/bswap.h
index 9a03a91..1190c73 100644
--- a/qtopia/src/3rdparty/libraries/libavcodec/bswap.h
+++ b/qtopia/src/3rdparty/libraries/libavcodec/bswap.h
@@ -157,6 +157,10 @@ #endif	/* !HAVE_BYTESWAP_H */
 // be2me ... BigEndian to MachineEndian
 // le2me ... LittleEndian to MachineEndian
 
+#ifdef __PPC__
+#define WORDS_BIGENDIAN
+#endif
+
 #ifdef WORDS_BIGENDIAN
 #define be2me_16(x) (x)
 #define be2me_32(x) (x)
-- 
1.3.3

